src/ppx_deriving_yojson_runtime.ml

```diff
@@ -8,7 +8,9 @@ let (>|=) x f =
 
 let rec map_bind f acc xs =
   match xs with
-  | x :: xs -> f x >>= fun x -> map_bind f (x :: acc) xs
+  | x :: xs -> (match f x with
```

Comment:
Could you include a comment to point out that you don't use `>>=` to keep the function tail-recursive under js-of-ocaml?